packages/core/nuxt-theme-module/theme/components/AppHeader.vue

```diff
@@ -12,8 +12,30 @@
         </nuxt-link>
       </template>
       <template #navigation>
-        <SfHeaderNavigationItem class="nav-item" data-cy="app-header-url_women" label="WOMEN" :link="localePath('/c/women')"/>
-        <SfHeaderNavigationItem class="nav-item"  data-cy="app-header-url_men" label="MEN" :link="localePath('/c/men')" />
+        <SfHeaderNavigationItem class="nav-item" data-cy="app-header-url_women" label="WOMEN" :link="localePath('/c/women')">
+          <!-- <template name="desktop-navigation-item">
+            <SfLink class="sf-header-navigation-item__link" :link="link">{{
+              label
+            }}</SfLink>
+          </template> -->
+
+          <template #mobile-navigation-item>
+            <SfMenuItem label="WOMEN" class="sf-header-navigation-item__menu-item">
+              <template #mobile-nav-icon>
+                &#8203;
+              </template>
+            </SfMenuItem>
+          </template>
+        </SfHeaderNavigationItem>
+        <SfHeaderNavigationItem class="nav-item"  data-cy="app-header-url_men" label="MEN" :link="localePath('/c/men')" >
+          <template #mobile-navigation-item>
+            <SfMenuItem label="MEN" class="sf-header-navigation-item__menu-item">
+              <template #mobile-nav-icon>
+                &#8203;
```

Comment:
do we really need to do such a hack? I think that should be fixed in SFUI

Reply:
yes for sure, but it is temporary solution to make it work